src/lib/select/select.html

```diff
@@ -29,16 +44,9 @@
   (backdropClick)="close()"
   (attach)="_onAttached()"
   (detach)="close()">
-
-  <div
-    #panel
-    class="mat-select-panel {{ _getPanelTheme() }}"
-    [ngClass]="panelClass"
-    [@transformPanel]="multiple ? 'showing-multiple' : 'showing'"
-    (@transformPanel.done)="_panelDoneAnimatingStream.next($event.toState)"
-    [style.transformOrigin]="_transformOrigin"
-    [style.font-size.px]="_triggerFontSize"
-    (keydown)="_handleKeydown($event)">
-    <ng-content></ng-content>
-  </div>
+  <ng-container *ngIf="!_inlineRendering" [ngTemplateOutlet]="panelTemplate"></ng-container>
 </ng-template>
+
+<div *ngIf="_inlineRendering" style="display: none;" #domPortalContent>
```

Comment:
Wouldn't the `display: none` here always be hiding the options from assistive technology?

Reply:
Nevermind, I realized that is is only to capture the content as soon as I submitted it.